proto/waves/invoke_script_result.proto

```diff
@@ -40,6 +40,16 @@ message InvokeScriptResult {
         Amount min_fee = 1;
     }
 
+    message Lease {
+        bytes recipient = 1;
```

Comment:
What is `recipient` here? Is it "address' body" like in `Payment`? Actually action itself can contain Address or Alias. Why not to use `Recipient` type from `recipient.proto`?

Reply:
Done